pages/users.tsx

```diff
@@ -0,0 +1,88 @@
+import type { NextPage } from 'next'
```

Comment:
this page seems to be missing the actions on the RHS, filtering and sorting for columns and bolding of column headers